files-csharp/FileAlreadyInUseException/FileAlreadyInUseExceptionTests.cs/ProgramTests.cs

```diff
@@ -0,0 +1,53 @@
+using FileAlreadyInUseException;
+using Xunit;
+
+namespace FileAlreadyInUseExceptionTests;
+
+public class ProgramTests
+{
+    [Fact]
+    public void WhenFileNotInUse_AndIsFileInUseGeneric_ThenReturnsFalse()
+    {
+        // Arrange
+        var filePath = "testFile0.txt";
+        using (File.Create(filePath)) ;
+
+        // Act
+        bool result = Program.IsFileInUseGeneric(new FileInfo(filePath));
+
+        // Assert
+        Assert.False(result);
+        File.Delete(filePath);
```

Comment:
Delete calls should move above the Assert so we don't leak a file in the case of test failure.